### ISSUE TITLE
Would it be possible to add this to HACS? (or native HA?)

### ISSUE THREAD

**Issue by pvmil** [lukevink/home-assistant-buienradar-forecast-card#13]
**Thanks for creating this addon!**

Ideal solution would be to add this to the existing (part of HA core) buienradar plugin (retrieving data only once?)
Adding to HA core or HACS would already help end possibly extend the userbase

**Comment by pvmil** [comment]
Sorry, duplicate of #2
